app/assets/stylesheets/styleguide/_typography.scss

```diff
@@ -36,6 +36,7 @@
 // CORE FONTS - NEW TRANSPORT
 
 @mixin core-48($line-height: (50 / 48), $line-height-640: (30 / 25)) {
+  // scss-lint:disable NameFormat
```

Comment:
Yeah, chatted with @dsingleton about this and concluded that this app would need some serious frontend modernising at some point!